recipes/_install.rb

```diff
@@ -0,0 +1,84 @@
+#
+# Cookbook Name:: copperegg
+# Recipe:: _install
+#
+# Copyright 2013-2016 IDERA
+# License:: MIT License
+#
+
+apikey = node['copperegg']['apikey']
+copperegg_url = node['copperegg']['url']
+is_windows = (node['platform_family'] == 'windows')
+
+directory node['copperegg']['config_dir'] do
+  if is_windows
+    owner 'Administrators'
+    rights :full_control, 'Administrators'
+    inherits false
+  else
+    owner 'root'
+    group 'root'
+    mode 0755
+  end
+  action :create
+  not_if { node['copperegg']['uninstall_collector'] }
+end
+
+agent_config_file = node['copperegg']['config_file']
+template agent_config_file do
+  if is_windows
+    owner 'Administrators'
+    rights :full_control, 'Administrators'
+    inherits false
+  else
+    owner 'root'
+    group 'root'
+    mode 0664
+    action :create
+  end
+  source 'copperegg.conf.erb'
+  variables(
+    :apikey => apikey,
+    :url => copperegg_url
+  )
+  not_if { node['copperegg']['uninstall_collector'] }
+end
+
+
+if is_windows
+  service 'RevealCloud' do
+    action :enable
+    only_if { File.exists?(agent_config_file) }
+  end
+
+  service 'RevealCloud' do
+    action :start
+    only_if { File.exists?(agent_config_file) }
+  end
+
+else
+  service 'revealcloud' do
+    supports :restart => true, :status => true, :start => true, :stop => true
+    action [:stop, :start]
+    only_if { File.exists?(agent_config_file) }
+  end
+
+end
+
+unless is_windows
+  if node['copperegg']['create_sshprobe'] && node.attribute?('ec2') && node.attribute.ec2.attribute?('public_hostname')
```

Comment:
did we mention in README about our new SSH probe feature for  the probe which we are creating here ? If not, please update